SampleScripts/PowerManagement-WorkloadDomain.ps1

```diff
@@ -254,6 +254,42 @@ Try {
         [Array]$vcfvms = @()
         [Array]$vcfvms += ($vcServer.fqdn).Split(".")[0]
 
+        # Gather VxRail Manager Details for the VI Workload Domain, if exists
```

Comment:
```suggestion
        # Gather VxRail Manager details for the VI workload domain, if it exists.
```

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108